resolwe_bio/docker_images/dnaseq/packages-manual/bamclipper.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+. /var/cache/build/packages-manual/common.sh
+
+download_and_verify \
+    genialis \
```

Comment:
Bamclipper was obtained at https://github.com/tommyau/bamclipper. We should reference that and maybe install this to `tommyau` folder instead of `genialis`

Reply:
Done.